demo/llm.rag.service/chat-serveragllmpluslb.yaml

```diff
@@ -15,11 +15,13 @@ spec:
       labels:
         model: serveragllm
       annotations:
-        node.elotl.co/instance-type-regexp: "^t3.xlarge$"
+        # node.elotl.co/instance-type-regexp: "^t3.xlarge$"
+        # On GKE use:
+        node.elotl.co/instance-type-regexp: "^n2-standard-4$"
```

Comment:
Should we try to create a regular expression that will match both EKS and GKE desired instance?
BTW, I'm thinking that our trial customer would be using EKS?

Reply:
that's a good idea. This way we don't need to have both and play with commenting out.

Reply:
I guess this should work 
```
^(t3\.xlarge|n2-standard-4)$
```
?

Reply:
changed